english/watch-movie/Up.md

```diff
@@ -1 +1,2 @@
-You decided to asked the marshmallow to watch your favourite movie Up with you.
\ No newline at end of file
+You decided to asked the marshmallow to watch your favourite movie Up with you.
+Halfway through the movie, the marshmallow walks out.  You change the TV to F-Troop and it comes back in and sits down.  Turns out it loves Agarn!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.